# Problem set 3

[Carmen Salazar]

[GSS 2018 Dataset] 

[3/4/2020]

In [39]:
*cd "/Users/karminsalazar/Desktop/Gradstats/gradstats2/week7"

*set more off

*capture log close

*log using problem_set_3log20200308.log

*use realrinc age sex race educ yearsjob paeduc PASEI10 using GSS2018w7.dta, clear

*describe

*summarize realrinc age sex race educ yearsjob paeduc PASEI10

## 2. Conduct statistical tests for your hypothesized relationship: [3 pts]

* Estimate at least 2 different types of non-linear models that test your hypotheses.

* Use appropriate regression and graphical representations to represent each model.

* Use esttab to compare between types of non-linear and linear models

In [27]:
est clear
eststo: reg realrinc race
eststo: reg educ sex




      Source |       SS           df       MS      Number of obs   =     1,363
-------------+----------------------------------   F(1, 1361)      =     23.30
       Model |  1.9120e+10         1  1.9120e+10   Prob > F        =    0.0000
    Residual |  1.1170e+12     1,361   820705801   R-squared       =    0.0168
-------------+----------------------------------   Adj R-squared   =    0.0161
       Total |  1.1361e+12     1,362   834141678   Root MSE        =     28648

------------------------------------------------------------------------------
    realrinc |      Coef.   Std. Err.      t    P>|t|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
        race |  -5354.918   1109.426    -4.83   0.000    -7531.289   -3178.547
       _cons |   32568.86   1750.679    18.60   0.000     29134.54    36003.19
------------------------------------------------------------------------------
(est1 stored)


      Source |       SS         

In [34]:
%html
esttab, stats(r2 N, labels("R-Sqaured" "N")) cells(b(star fmt(3)) se(fmt(3) par)) /// 
  nobase mlabels ("Model 1" "Model 2" "Model 3") starlevels(* .05 ** .01 *** .001) ///
  coeflabels (realrinc "Respondent's income in constant $" race "Race of respondent" educ "Highest year of school completed" sex "Respondents sex") html

This front-end or document format cannot display HTML

## 3. Interpret the results [1 pt]

    A. Say whether the results support or contradict your hypothesis and why
    
    B. Say whether a linear or particular non-linear model best describes your data and why

A. The results presented above supports my first hypothesis and contrdict my second
hypothesis because as seen above, the regression coefficient in model 1 of -5354.918 indicates 
a negative relationship. A negative relationship indicates that the dependent variable of income 
in constant dollars (realrinc) increases as the independent variable (race) decreases, or in 
this case, changes categorically from white to black. The regression coefficient in model 2 of 
0.071 indicates a positive relationship. A positive relationship indicates that the dependent 
variable of Highest year of school completed increases as the independent variable (sex) 
increases, or in this case, changes categorically from female to male.

B. I think a nominal non-linear model best describes my data because it is a model that allows a 
more accurate description of how the association between my variables varies by sex, race, income,
and education. It better describes the relationship between independent and dependent categorical
variables of interest.